docker-compose.yaml

```diff
@@ -1,24 +1,37 @@
-version: "3.3"
+########################### COMMON SETTINGS
+
+x-common-default: &common-default
+    security_opt:
+        - no-new-privileges:true
```

Comment:
Is this option necessary? I have not seen this before. What exactly does it do?

Reply:
While it's not strictly necessary, I believe it's a good practice to include the option no-new-privileges:true. This setting ensures that the container cannot gain additional capabilities beyond those it was initially granted, which enhances security. Additionally, Screensy has a TURN (coturn) server running with host network_mode so i believe that its a good flag to enable.